### PR TITLE
Getting the prerelase package version when stable version is not found

### PR DESCRIPTION
**Pull request by jinujoseph**
I am yet to test this change and will do before even I think of merging.
let me know if this looks like a reasonable solution.

### CODE REVIEW

**Comment by jinujoseph** [comment]
@rtaket

File: src/Ng/Models/RegistrationIndex.cs [hunk 1]
```diff
@@ -124,6 +124,67 @@ public RegistrationIndexPackage GetLatestStableVersion()
...
+                        if (package.PackageContent != null)
+                        {
+                            return package;
+                        }
+                    }
+                }
+            }
+            return null;
+        }
+
```
Comment (rtaket) [inline src/Ng/Models/RegistrationIndex.cs:187]: lgtm. 
If you need a machine to test this on, you can use mikefan-server0. Just overwrite the files at \mikefan-server0\Shares\NG\bin. Then log into the machine and run \mikefan-server0\Shares\NG\run.cmd.

Comment (rtaket) [inline src/Ng/Models/RegistrationIndex.cs:187]: Can you add a few prerelease packages to the list of required packages in the config file?
At the same time, can you add "MicrosoftAssemblyReferences" too?

Comment (jinujoseph) [inline src/Ng/Models/RegistrationIndex.cs:187]: done

**Pull request merged**